Static Analysis workflow failing

**Describe the bug**
It seems like after the release of GB 9.7 earlier today, the Static Analysis workflow fails for newly created PRs because of this: https://github.com/WordPress/gutenberg/blob/637c43de571b5648e867f3d4d223712895304041/packages/interface/src/components/interface-skeleton/index.js#L73-L77

The error is:

```
 75:4  error  Deprecated functions must be removed before releasing this version  no-restricted-syntax
```

Discovered here: https://github.com/WordPress/gutenberg/pull/28008#issuecomment-755610016

**To reproduce**

Make sure you're local clone of the repo is up-to-date. Create a new branch with some changes and file a PR. Note that the Static Analysis GH action workflow fails.

**Expected behavior**
That workflow to pass.

cc/ @JustinSainton

--- a/packages/interface/CHANGELOG.md
+++ b/packages/interface/CHANGELOG.md
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+### Breaking Changes
+
+-   `leftSidebar` prop in `InterfaceSkeleton` component was removed ([#26517](https://github.com/WordPress/gutenberg/pull/26517). Use `secondarySidebar` prop instead.
+
 ## 0.11.0 (2020-12-17)
 
 ### New Feature

--- a/packages/interface/src/components/interface-skeleton/index.js
+++ b/packages/interface/src/components/interface-skeleton/index.js
@@ -8,7 +8,6 @@ import classnames from 'classnames';
  */
 import { forwardRef, useEffect, useRef } from '@wordpress/element';
 import { __unstableUseNavigateRegions as useNavigateRegions } from '@wordpress/components';
-import deprecated from '@wordpress/deprecated';
 import { __ } from '@wordpress/i18n';
 
 function useHTMLClass( className ) {
@@ -36,8 +35,6 @@ function InterfaceSkeleton(
 		actions,
 		labels,
 		className,
-		// Deprecated props.
-		leftSidebar,
 		shortcuts,
 	},
 	ref
@@ -68,15 +65,6 @@ function InterfaceSkeleton(
 	};
 
 	const mergedLabels = { ...defaultLabels, ...labels };
-
-	if ( leftSidebar ) {
-		deprecated( 'leftSidebar prop in InterfaceSkeleton component', {
-			alternative: 'secondarySidebar prop',
-			version: '9.7.0',
-			plugin: 'Gutenberg',
-		} );
-		secondarySidebar = leftSidebar;
-	}
 
 	return (
 		<div

